lib/pipeline.js

```diff
@@ -37,28 +35,6 @@ class PipelineModel extends BaseModel {
     }
 
     /**
-     * Format the scm url to include a branch and make case insensitive
-     * @method formatScmUrl
-     * @param  {String}     scmUrl Github scm url
-     * @return {String}            Lowercase scm url with branch name
-     */
-    formatScmUrl(scmUrl) {
```

Comment:
this method is actually used in the API
https://github.com/screwdriver-cd/screwdriver/blob/7756636b22ebe84600f4f5e591bbe3c49d1e5208/plugins/pipelines/create.js#L22